cachi2/core/package_managers/rpm/main.py

```diff
@@ -224,15 +224,11 @@ def _generate_sbom_components(files_metadata: dict[Path, Any]) -> list[Component
     return components
 
 
-def inject_files_post(*args: Any, **kwargs: Any) -> None:
+def inject_files_post(from_output_dir: Path, for_output_dir: Path, **kwargs: Any) -> None:
```

Comment:
Should we also change the corresponding function in `resolver.py` to match these types?